traffic.lua

```diff
@@ -297,20 +284,32 @@ function courseplay:findaiTrafficCollisionTrigger(vehicle)
 	end;
 
 	local ret = false
-
+	local index = nil
+	
 	if vehicle.aiTrafficCollisionTrigger == nil then
-		local index = vehicle.i3dMappings.aiCollisionTrigger --getXMLString(xmlFile, "vehicle.aiTrafficCollisionTrigger#index");
+		if vehicle.i3dMappings.aiCollisionTrigger then		-- standard colli definition
```

Comment:
can we put this in a function instead of copying it?

Reply:
Sorry, I do not understand what you mean. What is copied?
Have you seen the following lines?

Reply:
I see the same if construct in  CollisionDetector:findAiCollisionTrigger(object).

Reply:
Now I got you. Yes, but this is the legacy code and  CollisionDetector is the great new approach. Shall I put it somehow in one function?

Reply:
would be great

Reply:
put in vehicles.lua courseplay:findAiCollisionTrigger(vehicle)